chrome_app/vendor/mooltipass/device.js

```diff
@@ -179,24 +179,56 @@ mooltipass.device.queue = [];
 // Hash for command timeout to verify same command
 mooltipass.device.queueHash = null;
 
+// Variable to keep track of the interval for checkStatus
+mooltipass.device.interval = null;
 
+// If moolticute is available, favor it on top of the app
+mooltipass.device.usingMoolticute = false;
 /*********************************************************************************************************************/
 
 /**
+ * Check for websocket in a different thread
+ * 
+*/
+mooltipass.device.checkForMoolticute = function() {
+    var dummySocket = new ReconnectingWebSocket('ws://127.0.0.1:30035');
```

Comment:
I'm afraid that we can't do that: we do not have "socket" permission inside our manifest... 
get in touch with me.